app/assets/javascripts/mumuki_laboratory/application/assets-loader.js

```diff
@@ -0,0 +1,18 @@
+mumuki.assetsLoader = {
+  layout: {
+    onStartLoading: function () {
```

Comment:
I think those names are misleading, since they look like a way to register handlers, but it is the actual event. 

I'd rename it to e.g. `onLoadingStarted`